twifbeddit-app/backend/src/entrypoint.js

```diff
@@ -1,16 +1,26 @@
-
 exports.lambdaHandler = async (event, _) => {
-
     if (event.path == "/randnum" && event.httpMethod == "GET")
         return {
             'statusCode': 200,
             'body': JSON.stringify({
                 num: Math.random()
             })
         }
-   
-    return {
-        'statusCode': 200,
-        'body': JSON.stringify(event)
-    }
+
+        path = event.path.split('/');
```

Comment:
Small thing, but since the rest of this isn't under the if statement delete one indentation on lines 10-25